src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -23,15 +23,64 @@
 	<NcAppNavigation :aria-label="t('spreed', 'Conversation list')">
 		<div class="new-conversation"
 			:class="{ 'new-conversation--scrolled-down': !isScrolledToTop }">
-			<SearchBox v-model="searchText"
+			<SearchBox ref="searchbox"
+				:value.sync="searchText"
 				class="conversations-search"
+				:class="{'conversations-search--expanded': isFocused}"
 				:is-searching="isSearching"
+				@focus="setIsFocused"
+				@blur="setIsFocused"
 				@input="debounceFetchSearchResults"
 				@submit="onInputEnter"
 				@keydown.enter.native="handleEnter"
 				@abort-search="abortSearch" />
-			<NewGroupConversation v-if="canStartConversations" />
+
+			<!-- Options -->
+			<div class="options"
+				:class="{'hidden-visually': isFocused}">
+				<NcActions class="filter-actions"
```

Comment:
```suggestion
		<NcActions class="filter-actions"
		  :class="{'hidden-visually': isFocused}"
```

It's probably possible to move these styles to the NcActions now, without additional div wrapper